labelbox/schema/project.py

```diff
@@ -1172,34 +1173,16 @@ def move_data_rows_to_task(self, data_row_ids: List[str],
             timeout=180.0,
             experimental=True)["project"][method]["taskId"]
 
-        status = self._wait_for_task(task_id)
-        if status != "COMPLETE":
-            raise LabelboxError(f"Data rows were not moved successfully")
+        task = self._wait_for_task(task_id)
+        if task.status != "COMPLETE":
+            raise LabelboxError(f"Data rows were not moved successfully: " +
```

Comment:
What type of Task is this job? Currently, Task.errors only returns value for a `JSON Import` task